pkg/feature/serverless/resources.go

```diff
@@ -1,9 +1,17 @@
 package serverless
 
 import (
+	infrav1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/infrastructure/v1"
 	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/feature"
 )
 
 func ServingCertificateResource(f *feature.Feature) error {
```

Comment:
is the whole implemenation only applied when Serverless is enabled and only for kserve at the moment?

Reply:
Yes, only applied to Kserve for now